spec/features/ctc/portal_spec.rb

```diff
@@ -1,6 +1,6 @@
 require "rails_helper"
 
-RSpec.feature "CTC Intake", active_job: true do
+RSpec.feature "CTC Intake", :js, :active_job do
```

Comment:
If you don't want `:js` here (`:js` can introduce slowness or flakiness), you could do something like:

```
RSpec.feature "CTC Intake", :efile_security, :active_job do
```

and then change `spec/rails_helper.rb` to be something like

```
diff --git a/spec/rails_helper.rb b/spec/rails_helper.rb
index 15fafcd5a..76d2167de 100644
--- a/spec/rails_helper.rb
+++ b/spec/rails_helper.rb
@@ -138,7 +138,7 @@ RSpec.configure do |config|
       Capybara.page.current_window.resize_to(2000, 4000)
     end

-    unless Capybara.current_driver == Capybara.javascript_driver
+    if example.metadata[:efile_security] || Capybara.current_driver != Capybara.javascript_driver
       allow(Rails.application.config).to receive(:efile_security_information_for_testing).and_return(
         {
           device_id: "7BA1E530D6503F380F1496A47BEB6F33E40403D1",
```

(untested; just an idea)